src/http_client_async.zig

```diff
@@ -1000,23 +1026,10 @@ proxy_authorization: ?[]u8 = null,
 proxy_tunneling: bool = false,
 proxy_tunnel: ?ProxyTunnel = null,
 aborted: ?*std.atomic.Atomic(bool) = null,
+async_http_id: u32 = 0,
 
-pub fn init(
-    allocator: std.mem.Allocator,
-    method: Method,
-    url: URL,
-    header_entries: Headers.Entries,
-    header_buf: string,
-    signal: ?*std.atomic.Atomic(bool),
-) HTTPClient {
-    return HTTPClient{
-        .allocator = allocator,
-        .method = method,
-        .url = url,
-        .header_entries = header_entries,
-        .header_buf = header_buf,
-        .aborted = signal,
-    };
+pub fn init(allocator: std.mem.Allocator, method: Method, url: URL, header_entries: Headers.Entries, header_buf: string, signal: ?*std.atomic.Atomic(bool)) HTTPClient {
+    return HTTPClient{ .allocator = allocator, .method = method, .url = url, .header_entries = header_entries, .header_buf = header_buf, .aborted = signal };
```

Comment:
if you use `,` at the end, it will auto format more pretty

Reply:
the comma at the end did the trick thanks, fixed in https://github.com/oven-sh/bun/pull/2143/commits/77368b63a8bca0fa27d529556d156198aea87a38